barretenberg/cpp/src/barretenberg/ultra_honk/decider_verifier.cpp

```diff
@@ -58,32 +56,15 @@ template <typename Flavor> bool DeciderVerifier_<Flavor>::verify()
         return false;
     }
 
-    std::array<GroupElement, 2> pairing_points;
-    // TODO(https://github.com/AztecProtocol/barretenberg/issues/1109): Remove this hack once the verifier runs on
-    // Shplemini for all Ultra flavors
-    if constexpr (bb::IsAnyOf<Flavor, UltraKeccakFlavor>) {
-        const auto opening_claim = Shplemini::compute_batch_opening_claim(accumulator->verification_key->circuit_size,
-                                                                          commitments.get_unshifted(),
-                                                                          commitments.get_to_be_shifted(),
-                                                                          claimed_evaluations.get_unshifted(),
-                                                                          claimed_evaluations.get_shifted(),
-                                                                          multivariate_challenge,
-                                                                          Commitment::one(),
-                                                                          transcript);
-        pairing_points = PCS::reduce_verify_batch_opening_claim(opening_claim, transcript);
-    } else {
-        // Execute ZeroMorph rounds. See https://hackmd.io/dlf9xEwhTQyE3hiGbq4FsA?view for a complete description of the
-        // unrolled protocol.
-        const auto opening_claim = ZeroMorph::verify(accumulator->verification_key->circuit_size,
-                                                     commitments.get_unshifted(),
-                                                     commitments.get_to_be_shifted(),
-                                                     claimed_evaluations.get_unshifted(),
-                                                     claimed_evaluations.get_shifted(),
-                                                     multivariate_challenge,
-                                                     Commitment::one(),
-                                                     transcript);
-        pairing_points = PCS::reduce_verify(opening_claim, transcript);
-    }
+    const auto opening_claim = Shplemini::compute_batch_opening_claim(accumulator->verification_key->circuit_size,
```

Comment:
maybe make the return type explicit (in the upcoming PRs) so that no one is getting confused later

Reply:
will do :)